examples/turn_off_notifications.py

```diff
@@ -0,0 +1,108 @@
+import civis
```

Comment:
Could you add a short description of this script/how to use at the top of the file here? Most of the other scripts in this folder look like they have a description/instructions at the top, e.g. https://github.com/civisanalytics/platform-code-examples/blob/main/examples/email_reporting_python.py#L1